pyproject.toml

```diff
@@ -16,7 +16,7 @@ dependencies = [
   "jinja2<4",
   "loguru<1",
   "matplotlib<4",
-  "numpy<2",
+  "numpy~=2.0",
```

Comment:
the fix should be compatible with numpy<2 as well, so we could also leave this unpinned

Reply:
Sounds good